_inc/client/lib/plans/constants.js

```diff
@@ -57,6 +57,20 @@ export const JETPACK_MONTHLY_PLANS = [
 	PLAN_JETPACK_SECURITY_REALTIME_MONTHLY,
 	PLAN_JETPACK_COMPLETE_MONTHLY,
 ];
+export const JETPACK_OFFER_RESET_PLANS = [
```

Comment:
I wonder if this (and the other mentions of "offer reset") should be named differently. It makes sense now, but on the long run I don't know if "offer reset" will mean much, as we keep iterating on plans.

Reply:
You're right @jeherve. I think a better way of doing here could be to invert the logic and refer to legacy plans instead. I'll take a not to handle that in another PR.